### PR TITLE
check-password

### PR DESCRIPTION
**Pull request by joanna-konopacka**

### CODE REVIEW

File: src/checkPassword.test.js [hunk 1]
```diff
@@ -1,19 +1,56 @@
...
     expect(checkPassword).toBeInstanceOf(Function);
   });
 
   it(`should return boolean`, () => {
-
+    const result = checkPassword("Password1!");
+    expect(typeof result).toBe("boolean");
   });
 
   it(`should return 'true' for the valid password with 8 characters`, () => {
```
Comment (vlasiuksasha) [inline src/checkPassword.test.js:15]: The test case for a valid password does not check for a password with exactly 8 characters. Adjust the test case to check for a valid password with a length of 8 to ensure edge cases are covered.

File: src/checkPassword.test.js [hunk 2]
```diff
@@ -1,19 +1,56 @@
...
   });
 
-  // write more tests here
+  it(`should return 'false' for too short password`, () => {
+    const invalidPassword = "P@w1!";
+    const result = checkPassword(invalidPassword);
+    expect(result).toBe(false);
+  });
+
+  it(`should return 'false' for too long password`, () => {
```
Comment (vlasiuksasha) [inline src/checkPassword.test.js:45]: The test case for a too long password does not check for a password with exactly 17 characters. Adjust the test case to check for an invalid password with a length of 17 to ensure edge cases are covered.

**Review by vlasiuksasha — CHANGES_REQUESTED** [review]
check the comments

File: src/checkPassword.test.js (hunk 2 above)
Comment (joanna-konopacka) [reply src/checkPassword.test.js:45]: fixed

File: src/checkPassword.test.js (hunk 1 above)
Comment (joanna-konopacka) [reply src/checkPassword.test.js:15]: fixed

Comment (vlasiuksasha) [inline src/checkPassword.test.js:15]: This test should also check if the function returns 'true' for a valid password with 16 characters, which is the maximum length according to the rules.

File: src/checkPassword.test.js [hunk 3]
```diff
@@ -1,19 +1,56 @@
...
 
   it(`should return boolean`, () => {
-
+    const result = checkPassword("Password1!");
+    expect(typeof result).toBe("boolean");
   });
 
   it(`should return 'true' for the valid password with 8 characters`, () => {
+    const validPassword = "Passw0rd";
+    const result = checkPassword(validPassword);
```
Comment (vlasiuksasha) [inline src/checkPassword.test.js:17]: The password 'Passw0rd' doesn't contain any special character, so the test description is misleading. Either change the password to a valid one according to the rules or change the test description accordingly.

**Review by vlasiuksasha — CHANGES_REQUESTED** [review]
check the comments

**Review by vlasiuksasha — APPROVED** [review]
